docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+ARG VERSION
+
+FROM php:${VERSION}-cli-alpine
+
+WORKDIR /app
+
+RUN apk --no-cache add pcre-dev ${PHPIZE_DEPS} \
+  && pecl install xdebug \
+  && docker-php-ext-enable xdebug \
+  && apk del pcre-dev ${PHPIZE_DEPS}
+
+RUN docker-php-ext-install -j$(nproc) bcmath
+
+RUN curl -sS https://getcomposer.org/installer | \
+    php -- --install-dir=/usr/local/bin --filename=composer
+
+ENV XDEBUG_MODE=coverage
```

Comment:
and these lines are unnecessary here because without them the container more suited for usage in general

for instance 
`docker run --rm -ti -v $PWD:/app -w /app elegant-bro/arrayee:7.4 PHP -dzend_extension=xdebug.so -dxdebug.mode=coverage vendor/bin/phpunit && php check-coverage.php coverage.xml 100 && vendor/bin/ecs --level psr12 check src`

Reply:
what do you mean "usage in general"? is it really required for dev image and what are the cases?

Reply:
enabled xdebug brings huge overhead IMO we can avoid this

Reply:
Ok. I will rewrite it.